src/handlers.js

```diff
@@ -166,14 +192,26 @@ async function statusHandler(message) {
 async function jobHandler(message) {
   let context = this.context;
 
+  // Authenticating as installation.
+  try {
+    debug(`Authenticating as installation ${message.payload.installationID} 
+      to handle ${message.payload.details['event.type']}`);
```

Comment:
Here the installationId comes from the payload -- not from the routing key.  I think that's OK (we're assuming that only tc-github can publish to the pulse exchange, so nothing else can forge the message).  However, it does mean that we don't need the installationId in the routingKey.